src/vault.py

```diff
@@ -47,6 +48,15 @@ def is_sealed(self) -> bool:
         """Returns whether Vault is sealed."""
         return self._client.sys.is_sealed()
 
+    def wait_for_unseal(self) -> None:
+        """Wait for Vault to be unsealed."""
+        elapsed_time = 0
+        while self.is_sealed() and elapsed_time < 30:
+            logger.info("Vault is sealed, waiting for unseal")
+            time.sleep(5)
```

Comment:
5 seconds seems like a long time to wait between polls.

Reply:
Reduced to 2.